### ISSUE TITLE
Assessing backgrounds

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1392]
Summary of what needs to happen

1.  We absolutely need "controlled" background  to make it at all useful

**Comment by ValWood** [comment]
1. Examples

6378670e7a31c356 | pat1-114/pat1-114; homozygous diploid
6a9c8e26d513116c | pat1-114/pat1-114 homozygous diploid
 7169c40527631f2c | pat1-114/pat1-114; homozygous diploid
 1ebe4a1548de4aff | pat1-114/pat1-114 homozygous diploid
 94b54d8b1a931fb9 | pat1-114

2bfae567526bc0ec | ade6-704 ura4-D18 leu1-32 h-
3bd00175a6e61e42 | leu1-32 ura4-D18 ade6-M216
67026591584779dd | leu1-32, ura4-D18, ade6-M216
86f72e4a0d568a17 | ade6- leu1-32 ura4-D18
c7dfdbd8467f6d60 | ade6-M210/ade6-M216, leu1-32/leu1-32, ura4-D18/ura4-D18 h+/h+
 c7dfdbd8467f6d60 | ade6-M210/ade6-M216, leu1-32/leu1-32, ura4-D19/ura4-D18 h+/h+

**Comment by ValWood** [comment]
2. Types of background

i.  tagged version of localized protein. Do we want to keep these? I have never captured, but if somebody has it isn't incorrect.  But we  wouldn't want to  give this it's own "genotype"?
examples:
 GFP-atb2
 GFP-Atb2
GFP Atb2 Cut11 GFP
 5b7414d8bc7f80eb | rlc1-GFP
tubulin CFP

ii. auxotrophic markers (the bulk)

iii. To create an experimental situation where a phenotype can be observed. 
i.e 
a) pat1-114 induce meiosis
see above
b) cdc25-22 block and release for synchronization
c) nda3-** to abolish the microtubule cytoskeleton
 nda3-KM311
nda3-KM
nda3

d) cdc10-*  to ?

4.  quite a  few WT ...should I remove these?
 nak1+ and orb6+ wild type product levels
nak1+ wild type product level
orb6+ wild type product level
ccq1+
d800c02f9eed68d9 | WT background

5. strain
h- 972
h-
h90

6. misc
f0afc6fe7b0b439a | slp1-shutoff  (should this be a knockdown?)

**Comment by ValWood** [comment]
should I:
remove commas
remove colons
remove tagged proteins
remove WT
standardize capitalization
move ploidy to "other"

**Comment by Antonialock** [comment]
remove WT - yes
remove tagged proteins - probably?

I'm not sure I follow 2. types of background. Do you see us moving from specifying a "flat list" background to categorized ones?
what if you don't know what a part of the background is relevant for?

**Comment by ValWood** [comment]
Here I was only tying to classify the different types of backgrounds we have used. 
This is to help us  assess how consistently we  as professional curators think we have specified these types  of background. This should help us to decide how we should capture.

At first glance:
- I think the community have probably supplied more background info than we have.
- I know that I have not consistently curated any of the types above , but if they are captured they are useful additional information (not necessarily for the phenotype in question, but for the use of the strain in  other contexts).

- On balance I think it would be best to store background as a "set of extensions" or "qualifiers" or  something attached to the genotype (we can call it "genotype extension" of "complete genotype" or whatever. 

We should probably not treat these as "individual genotypes" for display because of the  (major) inconsistency issues.
  
If we changed to make the background a piece of information attached to  the genotype it  would also make the backgrounds easier to capture during annotation (we would not need to create so may individual  genotypes).
( back ground + genotype =  complete genotype) we just  need to have appropriate labels.
